pallets/ddc-clusters/src/node_provider_auth.rs

```diff
@@ -50,6 +50,7 @@ where
 			None,
 			call_data,
 			false,
+			pallet_contracts::Determinism::Deterministic,
```

Comment:
what does this line do? Just of curiosity

Reply:
According to the code documentation in the 0.9.33 branch in substrate:
```
		/// - `determinism`: If this is set to any other value but [`Determinism::Deterministic`]
		///   then the only way to use this code is to delegate call into it from an offchain
		///   execution. Set to [`Determinism::Deterministic`] if in doubt.
```